README.md

```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
+|                     [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/AnthonyJCampbell)                      |                 [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/AbdelIdir)                 |                   [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/dueka)                   |
+| [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/anthonyjcampbell/) |         [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/idir-abdel/)         | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/ekanem-david-8711719a) |
 
-## License
+<br>
 
-This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
+|[Emma Andrews](https://github.com/ELAndrews)     | [John Afolabi](https://github.com/john-afolabi) |  [Karim Bertacche](https://github.com/john-afolabi)|  
+| :---------------------------------------------: | :---------------------------------------------: | :---------------------------------------------: |
+| [<img src="https://media-exp1.licdn.com/dms/image/C5603AQFRpv9tGUnasQ/profile-displayphoto-shrink_200_200/0?e=1587600000&v=beta&t=mN_NQjxyyVHRDkl0n-OpMoXj1qkYcuYGB5rQIPTBx7c" width = "200" />](https://github.com/ELAndrews) | [<img src="https://avatars3.githubusercontent.com/u/19263499?s=460&v=4" width = "200" />](https://github.com/john-afolabi) | [<img src="https://avatars3.githubusercontent.com/u/49835145?s=460&v=4" width = "200" />](https://github.com/KarimBertacche) |
+|                                                                                      [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/ELAndrews)                                                                                       |                                       [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/john-afolabi)                                       |                 [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/KarimBertacche)                  |
+|                                                                          [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/emma-andrews-b26768169/)                                                                           |                           [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/john-afolabi/)                           |       [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/karim-bertacche-64b270156/)       |
 
 ## Acknowledgments
 
 * Lambda 
```

Comment:
Perhaps we can add a link to Lambda School (https://lambdaschool.com) & a link to Alex (in his position as Engineering Manager)

Reply:
Completed